client/Api.ts

```diff
@@ -6329,7 +6606,7 @@ export class Api extends HttpClient {
      * Get BGP peer status
      */
     networkingBgpStatus: (_: EmptyObj, params: FetchParams = {}) => {
-      return this.request<void>({
+      return this.request<BgpPeerStatus[]>({
```

Comment:
This is what the array handling fix was. Before the changes in `generator/client/api.ts` and elsewhere, these would fall back to `void` because we did not know how to pull out an array type from the response schema.